pennylane/operation.py

```diff
@@ -1406,7 +1406,7 @@ def __add__(self, other):
             return qml.op_sum(self, other)
         if other == 0:
             return self
-        if isinstance(other, qml.ops.ParametrizedHamiltonian):  # pylint: disable=no-member
+        if isinstance(other, qml.pulse.ParametrizedHamiltonian):  # pylint: disable=no-member
```

Comment:
```suggestion
        if isinstance(other, qml.pulse.ParametrizedHamiltonian):
```
I think this can go then, right?